src/seaport.ts

```diff
@@ -2261,6 +2270,7 @@ export class OpenSeaPort {
       takerProtocolFee,
       makerReferrerFee,
       waitingForBestCounterOrder: waitForHighestBid,
+      englishAuctionReservePrice: makeBigNumber(englishAuctionReservePrice || 0),
```

Comment:
`orderToJSON` and `OrderType` also need to be updated, to send the data in our POST request